intake/cli/server/server.py

```diff
@@ -319,7 +320,27 @@ def post(self):
                                      metadata=source.metadata))
                 self.write(msgpack.packb(response, **pack_kwargs))
                 self.finish()
-            elif direct_access == 'force' and not client_has_plugin:
+            elif direct_access == 'signed' and client_has_plugin:
+                response = open_desc
+                user_parameters['plugin'] = plugin_name
+                response['args'] = entry._entry._create_open_args(user_parameters)[1]
+                response_args = response['args']
+                fs, *_ = get_fs_token_paths(response_args['urlpath'],
+                                            storage_options=response_args.get('storage_options', {}))
```

Comment:
The source storage options should be rewritten to whatever it takes to be able to read the HTTP URL, perhaps just {}